src/hotspot/cpu/aarch64/templateInterpreterGenerator_aarch64.cpp

```diff
@@ -820,7 +820,7 @@ void TemplateInterpreterGenerator::lock_method() {
   // get synchronization object
   {
     Label done;
-    __ ldrw(r0, access_flags);
+    __ load_unsigned_short(r0, access_flags);
```

Comment:
Could you please use `ldrh` rather than `load_unsigned_short` here? `load_unsigned_short` is only used in the termplate interpreter, and is a hangover from the hand-translation from x86.

Reply:
Oh, I thought it was quite nice that I didn't have to know the ldrh instruction as a platform independent load_unsigned_short was available.  I can change it in the aarch64 code.